atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -319,8 +321,9 @@ def run_image_task(self, image_build_conf):
                 task_result = self.session.getTaskResult(task_id)
             except Exception as exc:
                 task_result = repr(exc)
-            raise RuntimeError('image task, {}, failed: {}'
-                               .format(task_id, task_result))
+            raise RuntimeError('image task failed. see {}/taskinfo?taskID={}    \n'
+                               'underlying task failure: {}'
+                               .format(self.koji_url, task_id, task_result))
```

Comment:
I don't think koji_url (which is hub url) maps directly to brew web link. Can you verify that the link generated is valid?

Reply:
[mlangsdorf@redhatnow rh-osbs]$ git grep koji_hub osbs-prod.conf
osbs-prod.conf:12:koji_hub = https://brewhub.stage.engineering.redhat.com/brewhub
osbs-prod.conf:37:koji_hub = http://brewhub.engineering.redhat.com/brewhub
osbs-prod.conf:58:koji_hub = https://brewhub.stage.engineering.redhat.com/brewhub
osbs-prod.conf:80:koji_hub = https://brewhub.stage.engineering.redhat.com/brewhub
osbs-prod.conf:104:koji_hub = http://brewhub.engineering.redhat.com/brewhub
osbs-prod.conf:132:# koji_hub = https://brewhub.stage.engineering.redhat.com/brewhub
osbs-prod.conf:164:# koji_hub = https://brewhub.stage.engineering.redhat.com/brewhub
osbs-prod.conf:198:# koji_hub = http://brewhub.engineering.redhat.com/brewhub
osbs-prod.conf:232:# koji_hub = http://brewhub.engineering.redhat.com/brewhub

it really looks to me like koji_hub is the brew web link.

Reply:
If I click on any of those links, I get:
```
Method Not Allowed
This is an XML-RPC server. Only POST requests are accepted.
```

If I append `/taskinfo?taskID=1` to it, I get the same error. The correct prefix is `https://brewweb.engineering.redhat.com/brew`

I'm not sure if we have the information in reactor-config-map to build the correct link.
We could either 1) not make this a link, or 2) enhance reactor-config-map to provide Koji web url which might be useful for other cases as well (email notification).

Reply:
Okay.
I thought the original request was to make it a link - the original error returned the taskID number but it wasn't obvious to the end user that it was a taskID.
Adding the necessary scaffolding for the koji web URL will be a pain. I'm going to ask in the original obs if just getting the taskID made obvious would be enough.